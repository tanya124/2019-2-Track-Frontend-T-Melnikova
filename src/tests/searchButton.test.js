import React from 'react';
import renderer from 'react-test-renderer';
import Save from '../components/Buttons/SaveButton';

it('renders correctly', () => {
	const tree = renderer.create(<Save saveData={jest.fn()} />).toJSON();
	expect(tree).toMatchSnapshot();
});
