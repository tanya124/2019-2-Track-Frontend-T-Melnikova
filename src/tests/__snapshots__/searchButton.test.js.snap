// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  className="css-9kqm1z"
  id="save"
  onClick={[Function]}
  src="checkmark.svg"
/>
`;
